src/Adapter/MSTest.CoreAdapter/MSTestSettings.cs

```diff
@@ -238,6 +245,16 @@ private static MSTestSettings ToSettings(XmlReader reader)
                                 break;
                             }
 
+                        case "ENABLEBASECLASSTESTMETHODSFROMOTHERASSEMBLIES":
```

Comment:
>ENABLEBASECLASSTESTMETHODSFROMOTHERASSEMBLIES [](start = 30, length = 45)

can we add tests for this too please? you would find them under MSTest.CoreAdapter.Unit.Tests -> MSTestSettingsTests.cs